Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust-g"
-edition = "2018"
-version = "0.8.0"
+edition = "2021"
+version = "1.0.0"
 authors = ["Bjorn Neergaard <bjorn@neersighted.com>"]
```

Comment:
i feel like we sould update this at some point

Reply:
what do you think we should put? Only neer and space could be considered the 'authors', unless we just generically put `/tg/station and friends <idksomeaddress@tgstation.com>` or whatever email you guys use

Reply:
i meant adding spacemaniac somewhere

is there a maintainers field too? you can put rust-g dev team there

also we have no emails

Reply:
> The optional authors field lists people or organizations that are considered the "authors" of the package. The exact meaning is open to interpretation — it may list the original or primary authors, current maintainers, or owners of the package. An optional email address may be included within angled brackets at the end of each author entry.